### ISSUE TITLE
[Bug] Anti Void Bug on 1.18

### ISSUE THREAD

**Issue by SussyBob** [MeteorDevelopment/meteor-client#1781]
### Describe the bug

Anti Void activated bellow Y=0

### Steps to reproduce

```bash
I was mining and at Y=-1 and the Anti Void activated so i cant go down :V
```

### Meteor Version

0.4.7 Dev Build Version - 1362

### Minecraft Version

1.18

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by Wide-Cat** [comment]
lol

**Comment by RedCarlos26** [comment]
That was sure lmao

**Comment by MineGame159** [comment]
Fixed in 3293fb24d7d21e09833c2ba2297be08c7840899b.
